.typedoc/typedoc-plugin-frontmatter.js

```diff
@@ -13,6 +13,11 @@ export function load(app) {
 
   app.renderer.on(PageEvent.END, (page) => {
     const frontmatterGlobals = app.options.getValue("frontmatterGlobals");
+
+    const model = page.model;
+    const title = model.name || "Embedded analytics SDK documentation";
```

Comment:
Let's move the `"Embedded analytics SDK documentation"` value to the `.typedoc/typedoc.config.mjs` as 
```
frontmatterGlobals: {
    layout: "docs-api",
    title: "Embedded analytics SDK documentation"
},
```

and do 
```
frontmatterGlobals.title = model.name || frontmatterGlobals.title;
```